package.json

```diff
@@ -28,7 +28,7 @@
     "clean-printable": "rimraf src/content/**/printable.md",
     "preclean": "run-s clean-dist clean-printable",
     "clean": "rimraf src/content/**/_*.md src/**/_*.json",
-    "start": "npm run clean-dist && npm run fetch && cross-env NODE_ENV=development webpack-dev-server --config webpack.dev.js --env.dev",
+    "start": "npm run clean-dist && cross-env NODE_ENV=development webpack-dev-server --config webpack.dev.js --env.dev",
```

Comment:
Why is this still highlighted as a change vs `master`? can you checkout this file from master we don't want changes here with this PR right?

Reply:
Yeah, I will do it. 
It is from the previous commit of this PR .